CMSgov/mint-app
# Easi 2505/create model plan error handling

# EASI-2505

## Changes and Description

- Create Model Plan now uses user Account information for creating a collaborator

This was to correct an issue that previously occurred if there was an issue getting OKTA information about a user while making a model plan. It would create in-complete data for the model plan. This is now not going to be an issue, as UserAccount information is retrieved earlier by the auth middleware, and the app should error if you can't get the contextual user account. There is no need to query CEDAR for email information here.


## How to test this change
1. Log into local mint with OKTA
2.  Create a model plan. 
3. Ensure that there are no errors when creating a model plan

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
